src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -229,8 +229,12 @@ def logTaskCompletedSQL(task):
     print("Logging task output to db", task.UUID)
     taskUUID = task.UUID.__str__()
     exitCode = task.results["exitCode"].__str__()
-    stdOut = task.results["stdOut"]
-    stdError = task.results["stdError"]
+    # truncate task stdout/stderror logged to the database
+    # (to prevent mysql server errors, refs. #10732) 
+    TASK_STDOUT_LIMIT = 16*1024*1024
+    TASK_STDERR_LIMIT = 16*1024*1024
+    stdOut = (task.results["stdOut"][:TASK_STDOUT_LIMIT] + '..') if len(task.results["stdOut"]) > TASK_STDOUT_LIMIT else task.results["stdOut"]
+    stdError = (task.results["stdError"][:TASK_STDERR_LIMIT] + '..') if len(task.results["stdError"]) > TASK_STDERR_LIMIT else task.results["stdError"]
```

Comment:
@jrwdunham @Hwesta any advice? I noticed stdOut and StdError are converted to unicode with strtoUnicode() after the truncation is done, should it be done before to address what @sevein pointed out?

I also have a question about the truncation limit size, This PR currently proposes to truncate at 16MB. As the purpose of these strings seem to be only to show the output of the command to the user in the dashboard, we may not even need that much? Maybe storing/showing say, the first and last MB of stdout and stderr could be more than enough?